actions/main/main.py

```diff
@@ -101,16 +102,16 @@ def main(params):
     similar_ids = model.similar_items(repo_ids[reference_repo])
     LOGGER.info('found %d similar repos' % len(similar_ids))
 
-    result = []
+    similar_repos = []
     for idx in range(1, len(similar_ids)):
-        result.append(repos[similar_ids[idx][0]])
+        similar_repos.append(repos[similar_ids[idx][0]])
 
-    return {'reference_repo': reference_repo, 'similar_repos': result, 'error': ''}
+    return {'reference_repo': reference_repo, 'similar_repos': similar_repos, 'error': ''}
 
 
 # for local testing
 if __name__ == "__main__":
-    # TODO: replace with click
```

Comment:
This seems to be the only change that is not an improvement but only to make pylint happy. 

But thats ok.